exercices/ana/core.clj

```diff
@@ -0,0 +1,74 @@
+(ns cipher.core
+  (:require [clojure.string :as string])
+  (import [java.lang Character]))
+
+; write caesar cipher
+
+(defn to-int [letter-char]
+  "takes a lowercase letter character and returns its position in the alphabet: a = 0, b = 1, etc."
+  (let [ascii-a (int \a)]
+    (- (int letter-char) ascii-a)))
+
+
+(defn to-char [position]
+  "takes an integer and returns the lowercase letter char in that int position: 0 = a, 1 = b, etc."
+  (let [ascii-a (int \a)]
+    (char (+ position ascii-a))))
+
+
+(defn shift [letter n]
+  "takes a letter and an int and returns this letter shifted by n"
+  (-> (to-int letter)
+      (+ n)
+      (mod 26)
+      (to-char)))
```

Comment:
Here you don't need necessarily use `()`, you could replace `(to-char)` and use just `to-char`